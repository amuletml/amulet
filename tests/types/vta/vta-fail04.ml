let _ = (let 1 = 1 in fun x -> x) @int
