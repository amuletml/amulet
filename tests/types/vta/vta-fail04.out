vta-fail04.ml[1:9 ..1:38]: error
  Can not apply expression of type 'f -> 'f
    to visible type argument int

  Arising in the expression
  │ 
1 │ let _ = (let 1 = 1 in fun x -> x) @int
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
